paddle/fluid/inference/api/analysis_predictor.h

```diff
@@ -161,4 +181,62 @@ class AnalysisPredictor : public PaddlePredictor {
   bool status_ir_optim_enabled_{false};
 };
 
+class AnalysisPredictor::Quantizer {
```

Comment:
@Superjomn Could you help review this, could `class AnalysisPredictor::Quantizer` be put here?

Reply:
Please separate the quantizer related data structure to another file, keep it clean.
It is OK to add several methods in AnalysisPredictor but put the main implementation in other modules.

Reply:
OK

Reply:
@Superjomn , we are trying to do that, but it is not easy due to quite a bit of dependencies.

Reply:
@Superjomn , done in https://github.com/PaddlePaddle/Paddle/pull/16396